libs/core/src/lib/form/form-control/form-control.directive.ts

```diff
@@ -34,6 +34,9 @@ export class FormControlDirective extends AbstractFdNgxClass {
     @Input()
     type: string;
 
+    /** @hidden */
+    elementRef: ElementRef;
+
```

Comment:
If you need to use it somewhere else, maybe put it to `constructor(public elementRef: ElementRef) {}`